SAP2000_Engine/Query/SplitRigidLink.cs

```diff
@@ -36,24 +36,24 @@ public static partial class Query
 
         public static List<RigidLink> SplitRigidLink(RigidLink link)
         {
-            List<RigidLink> links = null;
+            List<RigidLink> SingleLinks = new List<RigidLink>();
```

Comment:
minor comment, but should be lowercase letter

Reply:
Yep, you're right!